src/Assessor/AssessorPermissions.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace AppBundle\Assessor;
+
+use MyCLabs\Enum\Enum;
+
+class AssessorPermissions extends Enum
+{
+    public const MANAGE = 'MANAGE';
```

Comment:
IMO you can move this const in the voter

Reply:
Ok :)